docs/distribution/README.md

```diff
@@ -0,0 +1,923 @@
+# OCI Distribution - Notary v2 Signature Support
+
+To support [Notary v2 goals][notaryv2-goals], upload, persistance and discovery of signatures must be supported.
+
+To minimize the complexity of registry operators and projects to adopt Notary v2, a balance between leveraging what already exists and new patterns to support secure discovery are explored.
+
+## Table of Contents
+
+* [Signature Persistance](#signature-persistance)
+* [Signature Push](#signature-push)
+* [Signature Discovery](#signature-discovery)
+* [Signature Pull](#signature-pull)
+* [Example Artifacts](#example-artifacts)
+
+## Signature Persistance
+
+Several options for how to persist a signature were explored. We measure these options against the [goals of Notary v2][notaryv2-goals], specifically:
+
+* Maintain the original artifact digest and collection of associated tags, supporting existing dev through deployment workflows
+* Multiple signatures per artifact, enabling the originating vendor signature, public registry certification and user/environment signatures
+* Native persistance within an OCI Artifact enabled, distribution*spec based registry
+* Artifact and signature copying within and across OCI Artifact enabled, distribution spec based registries
+* Support multi-tenant registries enabling cloud providers and enterprises to support managed services at scale
+* Support private registries, where public content may be copied to, and new content originated within
+* Air-gapped environments, where the originating registry of content is not accessible
+
+To support the above requirements, signatures are stored as separate [OCI Artifacts][oci-artifacts]. They are maintained as any other artifact in a registry, supporting standard operations such as listing, deleting, garbage collection and any other content addressable operations within a registry.
+
+Following the [OCI Artifacts][oci-artifacts] design, signatures are identified with: `config.mediaType: "application/vnd.cncf.notary.config.v2+jwt"`.
+The config object contains the signature and signed content. See [nv2-signature-spec][nv2-signature-spec] for details.
+
+Storing a signature as a separate artifact enables the above goals, most importantly the ability to maintain the existing tag and and digest for a given artifact.
+
```

Comment:
In terms of actual deployment, do we want to explicitly call out a key mgmt or vault interactions? Definition of air-gapped may also mean even a vault may not be trusted, etc. This part may or may not be outside scope of this document.

Reply:
Yes, definitely, but we're looking to the key management working group to make the first round of recommendations for what and how.